app/policies/subject_rule_effect_policy.rb

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Remove blank line at beginning of file

Reply:
i'm not sure why this doesn't show up as outdated like the other changes i incorporated, but it definitely is gone :)